prepare/metrics/rag.py

```diff
@@ -416,3 +416,144 @@
     add_to_catalog(
         metric, f"metrics.rag.response_generation.{axis}.{base_metric}", overwrite=True
     )
+
+#############################################
+#### End to end rag metrics #################
+#############################################
+
+end_to_end_artifact_names = [
+    TaskRagEndToEndConstants.METRICS_RAG_END_TO_END_ANSWER_CORRECTNESS,
```

Comment:
Can these names be shortened?

For example, consider:
`RagEndToEnd.ANSWER_CORRECTNESS`
so `RagEndToEnd` is not repeated twice, being a constant is implied from the use of `UPPER_CASE`, `Task` is implied from `RagEndToEnd` and `Metric` is implied from `ANSWER_CORRECTNESS`.

Would makes things much more readable IMO.. WDYT?